docs/privacy-policy.md

```diff
@@ -1,178 +1,157 @@
 
-- We store username, email address and social media usernames. Username and email address are mandatory.
-- Credentials. We store passwords. All of the Credentials are hashed and original values not stored.
-- All personal information that you provide to us must be true, complete and accurate, and you must notify us of any changes to such personal information.
+## How do we collect your data?
 
-### **Information automatically stored**  
+All of the data is directly provided by your activity on the Site. The data is
+processed when you:
 
-_In Short: Certain information – such as IP address and/or browser and device characteristics – is stored automatically when you visit our websites._  
+- Register a new account
+- Visit userstyle details page (e.g. [https://userstyles.world/style/1/userstyles-world-tweaks][details])
+- Visit userstyle install page (e.g. [https://userstyles.world/api/style/1.user.css][api])
 
-We automatically store certain information when you visit, use or navigate the Site. This information does not reveal your specific identity (like your name or contact information) but may include device and usage information, such as your IP address, browser and device characteristics, operating system, information about how and when you use our Site and other technical information. This information is primarily needed to maintain the security and operation of our Site, and for our internal analytic and reporting purposes.  
+[details]: https://userstyles.world/style/1/userstyles-world-tweaks
+[api]: https://userstyles.world/api/style/1.user.css
 
-We'd like to state that there are 2 different automatic storing mechanisms:
 
-The first one is default server log which contains this information, and is flushed every 24 hours.
-This is primarily needed for security and maintaining the operation of the Site.
+## How do we use your data?
 
-The second one is used by style statistics and only happens on certain portions of the site:
+We use your data in order to show userstyle statistics.
 
-- `https://userstyles.world/style/:id/:name`
-- `https://userstyles.world/api/style/:id.user.css`
 
-By visiting the relative links stated above, we save a unique hash combined of your IP address, the style ID, reffered within the relative link as `:id`, and a secret key. This unique hash is not easily reversible without brute-forcing all public IP addresses in IPv4 address space and our secret key. This provides decently accurate style statistics while still respecting your privacy. 
+## How do we store your data?
 
-## HOW DO WE USE YOUR INFORMATION? 
+We follow best practices when it comes to handling your data and store
+everything on our own server.
 
-_In Short: We process your information for purposes based on legitimate business interests, the fulfillment of our contract with you, compliance with our legal obligations, and/or your consent._  
+The data used for userstyle statistics is stored as a unique hash, and is not
```

Comment:
For some reason in this sentence I feel like the need of "anonymizing the IP trough an hash that's combined with a secret key"

Reply:
That is exactly what I was thinking of.